packages/react-composites/src/composites/ChatComposite/adapter/AzureCommunicationChatAdapter.ts

```diff
@@ -418,7 +418,6 @@ export const createAzureCommunicationChatAdapter = async ({
   await chatClient.startRealtimeNotifications();
 
   const adapter = await createAzureCommunicationChatAdapterFromClient(chatClient, chatThreadClient);
-  await adapter.fetchInitialData();
```

Comment:
Yep, ideally this should be done by composite instead of adapter(And we assume in most case adapter will be used with composite) - if people are using adapter directly, they will need to simulate what composite is doing, including this fetchInitial data